packages/core/src/domain/configuration/endpointBuilder.ts

```diff
@@ -49,23 +56,22 @@ export function createEndpointBuilder(initConfiguration: InitConfiguration, buil
     useAlternateIntakeDomains,
   } = initConfiguration
 
-  function build(endpointType: EndpointType, source?: string) {
-    const host = buildHost(endpointType)
-    const path = buildPath(endpointType)
-    const queryParameters = buildQueryParameters(endpointType, source)
-    const endpoint = `https://${host}${path}?${queryParameters}`
+  const host = buildHost(endpointType)
+  const path = buildPath(endpointType)
 
+  function build(withBatchTime?: boolean): string {
```

Comment:
since `withBatchTime` is only used for rum, what about simplify the API and configure that only here?

Reply:
Yes, nice!